modules/service/src/main/scala/lucuma/odb/service/GmosLongSlitService.scala

```diff
@@ -38,11 +36,11 @@ trait GmosLongSlitService[F[_]] {
 
   def selectNorth(
     which: List[Observation.Id]
-  ): F[Map[Observation.Id, SourceProfile => GmosNorth]]
+  ): F[Map[Observation.Id, GmosNorth]]
```

Comment:
The `SourceProfile` was part of the binning calculation.  Since we're reading it from the table directly now, it is no longer needed.